spec/system/posts/index.spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts Index Page', type: :system do
```

Comment:
- [ ] Your test examples do not run at all. You have the wrong file names. Your file names should be `_spec.rb`, not `.spec.rb`.

  `index.spec.rb` - ❌
  `index_spec.rb` - ✔️ 
  `show.spec.rb` - ❌
  `show_spec.rb` - ✔️ 

  ![image](https://github.com/derrick1451/Blog-app/assets/54848363/d1d3cbc1-5421-4dbd-8e13-89836c664780)